formValidation.js

```
@@ -0,0 +1,15 @@
+const email = document.getElementById('email');
+const msg = document.querySelector('small');
+const form = document.getElementById('submitbtn');
+
+email.addEventListener('submit', (event) => {
+  if (email.value !== email.value.toLowerCase()) {
+    msg.textContent = 'Please use lower case only.';
+    email.style.border = '1px solid red';
+    event.preventDefault();
```

- [ ] Kindly fix the error because when the user submits the form that contains email in uppercase then It is submitting so, According to required the form should submit when the email is in lowercase otherwise It should throw an error as a message `Please use lower case only`.